packages/apollo-link-batch-http/package.json

```diff
@@ -52,7 +51,7 @@
     "object-to-querystring": "1.0.8",
     "proxyquire": "1.8.0",
     "rimraf": "2.6.3",
-    "rollup": "0.68.2",
+    "rollup": "^1.4.1",
```

Comment:
Won't the upgrade-bot complain about ^?

Reply:
Ahh good point. Pinned: 4447dc99e0820614ba8eee11f1ea3b5ef7a78960